.github/workflows/codeql-analysis.yml

```diff
@@ -6,9 +6,6 @@
 name: "Static Application security Testing (CodeQL)"
 
 on:
-  push:
-    branches:
-      - "*"
```

Comment:
Instead of deleting this, let's just ignore push events to Dependabot PRs: 
```
push:
  branches-ignore:
    - "dependabot/**"
```
(based on https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on)